index.html

```diff
@@ -248,6 +248,23 @@ <h2>Conventions used in this document</h2>
     <section id="resource-management">
       <h2>Resource Management</h2>
 
+      <section id="resource-management-intro" class="informative">
+        <h2>Introduction</h2>
+        <p>
+          This section defines the core operations for managing content, using [[!LDP]] as a baseline. It
+          extends [[!LDP]] in several ways, including making some optional features mandatory and expanding
+          requirements related to binary content.
+        </p>
+        <p>
+          [[!LDP]] defines a hierarchy of resources, with a top-level distinction between RDF resources
+          (<a>LDP-RS</a>) and non-RDF resources (<a>LDP-NR</a>).  RDF resources are further divided into
```

Comment:
I think we should make it clear that not all RDF resources are containers. May also be easier to split into two sentence, something like:

> RDF resources may function as containers (<a>LDPC</a>) of other resources. There are several container subtypes, which are primarily differentiated by how much the container-member relationship can be configured.

Reply:
👍 just pushed an update to split into two sentences.